Makefile

```diff
@@ -179,11 +179,11 @@ unit:
 # Operator-sdk is smart enough to detect vendor directory
 .PHONY: run-ci-e2e-test
 run-ci-e2e-test:
-	hack/run-ci-e2e-test.sh -t basic
+	WMCO_VERSION=$(WMCO_VERSION) hack/run-ci-e2e-test.sh -t basic
 
 .PHONY: run-ci-e2e-upgrade-test
 run-ci-e2e-upgrade-test:
-	hack/run-ci-e2e-test.sh -t upgrade
+	WMCO_VERSION=$(WMCO_VERSION) hack/run-ci-e2e-test.sh -t upgrade
```

Comment:
What is going on here and why is this needed?

Reply:
Variables in the Makefile arent actually exported and therefore the variable needs to be explicitly set for the command